net/dns/resolver/forwarder.go

```diff
@@ -698,7 +696,17 @@ func (f *forwarder) forwardWithDestChan(ctx context.Context, query packet, respo
 		resolvers = f.resolvers(domain)
 		if len(resolvers) == 0 {
 			metricDNSFwdErrorNoUpstream.Add(1)
-			return errNoUpstreams
+			res, err := nxDomainResponse(query)
```

Comment:
It would be good to log something for Support to be able to easily see that there is a problem preventing DNS from working. It doesn't need to log a line for every query.

Reply:
There's already a metric that we can expose for that.

Reply:
We have the fleetwide metrics coalesced in dashboards. I don't know how to see metrics from an individual client. Do we have a reasonable way for support to look at this?